templates/alerts_logger.service.j2

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Tendrl alerts logger for user {{ tendrl_user }}
```

Comment:
Instead of using jija template and generate actual system unit file, you can use template feature of system uni files, see: https://fedoramagazine.org/systemd-template-unit-files/

That way, you don't need to edit/generate systemd unit file, and you can pass the parameter as a part of service name to start/stop.